File: src/main/java/bg/fmi/rateuni/dto/response/DisciplineResponse.java
package bg.fmi.rateuni.dto.response;

import bg.fmi.rateuni.vo.DisciplineCategory;
import bg.fmi.rateuni.vo.DisciplineType;
import lombok.Data;

import java.util.UUID;

@Data
public class DisciplineResponse {
    private UUID id;
    private String name;
    private double credits;
    private DisciplineCategory category;
    private DisciplineType type;
}

File: front-end/src/app/dto/response/discipline-response.ts
import {DisciplineCategory} from "../../vo/discipline-category";
import {DisciplineType} from "../../vo/discipline-type";

export class DisciplineResponse {
      id: string;
      name: string;
      credits: number;
      category: DisciplineCategory;
      type: DisciplineType;
      lecturer: string;
      assistants: string;
}
